extensions/stackdriver/edges/mesh_edges_service_client.cc

```diff
@@ -79,9 +79,11 @@ void MeshEdgesServiceClientImpl::reportTrafficAssertions(
   LOG_TRACE("mesh edge services client: sending request '" +
             request.DebugString() + "'");
 
-  context_->grpcSimpleCall(
-      grpc_service_, kMeshEdgesService, kReportTrafficAssertions, request,
-      kDefaultTimeoutMillisecond, success_callback_, failure_callback_);
+  HeaderStringPairs initial_metadata;
+  context_->grpcSimpleCall(grpc_service_, kMeshEdgesService,
```

Comment:
Is this because of an underlying API change?

Reply:
Yeah, the initial metadata callback was not usable until this.